.github/actions/release/action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Action: 🦋 Changesets Release'
+description: 'Release a new version of the package using changesets'
+
+runs:
+    using: "composite"
+
+    steps:
+        - name: Enter prerelease mode
+          shell: sh
+          if: ${{ github.ref != 'refs/heads/main' }}
+          run: yarn prerelease:start
+
+        - name: Exit prerelease mode
+          shell: sh
+          if: ${{ github.ref == 'refs/heads/main' }}
+          run: yarn prerelease:end
```

Comment:
While we do have a prerelase mode, we havent used it in recent times. Im considering to remove it since patches are very easy to put out.

Reply:
I though this one was for testing that the command would work or not before releasing it. Misunderstood the command. I will remove these scripts